dask_cuda/local_cuda_cluster.py

```diff
@@ -155,11 +162,12 @@ class LocalCUDACluster(LocalCluster):
 
     def __init__(
         self,
+        CUDA_VISIBLE_DEVICES=None,
         n_workers=None,
```

Comment:
Failures in `test_spill` are because `n_workers` is being passed by position. Could you fix them?

But also in the future we should prevent reordering kwargs for the same reason.

Reply:
Sure! Good point, I recognize that the reordering probably causes more harm than good here.